charmcraft/templates/init/src/charm.py.j2

```diff
@@ -31,9 +31,12 @@ class {{ class_name }}(CharmBase):
         if fail:
             event.fail(fail)
         else:
-            event.set_result(
-                "Probability is too important to be left to the experts."
-                " -- Richard Hamming")
+            event.set_results(
+                {
+                    "fortune": "Probability is too important to be left to the experts."
+                    " -- Richard Hamming"
```

Comment:
What an ugly indentation, it looks like there are two entries in the dict. May you indent the second line a little more to match the "value" start?

Reply:
better now?